File: Operators and Expression/PointInCurcle/PointInCircle.cs
using System;
//Problem 7. Point in a Circle
//Write an expression that checks if given point (x, y) is inside a circle K({0, 0}, 2).
class PointInCircle
{
    static void Main()
    {
        Console.WriteLine("Please intput coordinates X: ");
        double x = double.Parse(Console.ReadLine());

        Console.WriteLine("Please intput coordinates Y: ");
        double y = double.Parse(Console.ReadLine());

        if (x*x + y*y <= 4)
        {
            Console.WriteLine("True");
        }
        else
        {
            Console.WriteLine("False");
        }
    }
}

File: Javascript Fundamentals/OperatorsExpressions/js/PointInCircle.js
 
// Write an expression that checks if given point P(x, y) is within a circle K({0,0}, 5). 

var x=1;
var y = 2;
var radius = 3;

if ((x * x + y * y) <= (radius * radius)) {
	var result = "IS";
}
else {
	var result = "IS NOT";
}
console.log("The point with coordinates(X:" + x + " Y:" + y + ") "+ result +" within a circle with radius of "+ radius +"!");